src/nodeattr_interface.rb

```diff
@@ -35,7 +35,7 @@ def nodes_in_primary_group(primary_group)
         end.keys
       end
 
-      def nodes_in_group(group)
+      def nodes_in_gender(group)
```

Comment:
This change makes me think the other methods in this module should also be renamed to be consistent with our currrent usage of 'genders' vs 'groups' - what do you think about renaming the following:

- `nodes_in_primary_group` -> `nodes_in_group`;

- `groups_for_node` -> `genders_for_node`;

- `nodes_to_groups` -> `nodes_to_genders`?

Reply:
Updated